data.md

```diff
@@ -0,0 +1,106 @@
+
+In general, personal information you submit to us is used either to **respond to requests that you make, or to aid us in serving you better**.
+We do our best to protect the personal information you share with us, and do not share them with third parties.
+We may however, **display and share anonymized feedback or personal testimonials on our website and social media, or in other publicly viewable places**.
+
+### Where we keep data
+
+Our services are hosted on servers managed by third-party service providers, physically located in Canada.
+These servers are accessible to members of Hypha, and specific collaborators as deemed necessary to deliver the services and content.
+Where necessary, servers may also be accessed by other parties to comply with laws or to respond to lawful requests and legal process and in an emergency to protect the personal safety of any person.
+
+### How long we keep data
+
+System logs are generally retained for a duration that is appropriate for the particular system.
+
+User accounts necessary to deliver services and content are generally kept indefinitely, and periodically backed up, until the account holders have requested to delete them.
+Deleting your account on a particular service generally does not mean all your information is deleted from our periodic backups.
+If you have a concern, please email `data [at] hypha.coop`.
+
+Videoconferencing audio and video are only kept briefly in the buffer throughout the call and not recorded.
+
+Our [BigBlueButton server](https://bbb.hypha.coop) keeps a database for registered users and room information.
+Uploaded presentations are kept for 5 days after you have ended your meeting and log history is kept for 28 days.
+These are BigBlueButton's default configurations and automated purging is performed by the software.
+Response to polls, raise hand events, and chat messages are cleared after the call has ended.
+
+Our [Jitsi server](https://jitsi.hypha.coop) clears all meeting data in the room after the call has ended.
+
+The [Matrix chat](https://chat.tomesh.net/#/group/+hyphacoop:tomesh.net) in Hypha rooms are periodically wiped to keep only the most recent 3 months of history.
+However, as Matrix is a federated chat protocol, chat history older than 3 months may be retained by other homeservers, and in particular, public rooms with users from other homeservers.
+
+## Our Social Media Accounts and Third-party Services
+
+We use several social media accounts to share our work. We occasionally use the analytics tools provided by these platforms.
+Our social media accounts include:
+
+- [hyphacoop on Twitter](https://link.hypha.coop/twitter) ([privacy policy](https://twitter.com/en/privacy))
+  <li><a href="https://link.hypha.coop/linkedin" data-proofer-ignore="true" target="_blank">hyphacoop on LinkedIn</a> (<a href="https://www.linkedin.com/legal/privacy-policy" data-proofer-ignore="true" target="_blank">privacy policy</a>)</li>
```

Comment:
We cannot use md for the `<li>`?

Reply:
might be because despite being kramdown, gitbook doesn't process md in the same way jekyll does...

should test whether this works correctly regardless, might need whole list in html, sometimes mg doesn't render as expected alongside html